From: "K. Y. Srinivasan" <kys@microsoft.com>
Date: Thu, 19 Jan 2017 11:51:53 -0700
Patch-mainline: v4.11-rc1
Subject: Drivers: hv: vmbus: Get rid of an unsused variable
Git-commit: d383877db60bcc7fd02d1051a90e078d731dfb59
References: fate#320485

The version variable while it is initialized is not used;
get rid of it.

Signed-off-by: K. Y. Srinivasan <kys@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Acked-by: Olaf Hering <ohering@suse.de>
---
 drivers/hv/hv.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/hv/hv.c b/drivers/hv/hv.c
index ae5436e9c8a4..ced2077bb925 100644
--- a/drivers/hv/hv.c
+++ b/drivers/hv/hv.c
@@ -261,18 +261,14 @@ void hv_synic_free(void)
  */
 void hv_synic_init(void *arg)
 {
-	u64 version;
 	union hv_synic_simp simp;
 	union hv_synic_siefp siefp;
 	union hv_synic_sint shared_sint;
 	union hv_synic_scontrol sctrl;
 	u64 vp_index;
 
 	int cpu = smp_processor_id();
 
-	/* Check the version */
-	rdmsrl(HV_X64_MSR_SVERSION, version);
-
 	/* Setup the Synic's message page */
 	rdmsrl(HV_X64_MSR_SIMP, simp.as_uint64);
 	simp.simp_enabled = 1;
